Source/NSButton.m

```diff
@@ -543,4 +547,59 @@ - (NSSound *) sound
   return [_cell sound];
 }
 
+// Implement 10.7+ radio button behavior
+- (BOOL) _isRadioButton
+{
+  return ([[self cell] image] ==
+          [NSImage imageNamed: @"NSRadioButton"]);
+}
+
+- (void) _flipState: (NSButton *)b
+{
+  if ([b _isRadioButton])
+    {
+      if ([self action] == [b action] && b != self)
+        {
+          if ([self state] == NSOnState)
```

Comment:
As I already commented twice this statement could be taken out of the loop. Maybe check it on line 576 or close to that. Or did you mean to get the state from b? That is `if ([b state] == NSOnState)`? That would make sense as well, maybe check both?

Reply:
Removed as stated.   It is no longer needed.